libflux/src/flux/semantic/builtins.rs

```diff
@@ -275,6 +275,267 @@ pub fn builtins() -> Builtins<'static> {
+                    ) -> [t1]
+                "#),
+                "stddev" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        <-tables: [t0],
+                        ?column: string,
+                        mode: string
+                    ) -> [t1]
+                "#),
+                "string" => Node::Builtin("forall [t0] (v: t0) -> string"),
+                "sum" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        <-tables: [t0],
+                        ?column: string
+                    ) -> [t1]
+                "#),
+                "tableFind" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        <-tables: [t0],
+                        fn: (key: t1) -> bool
+                    ) -> t0
+                "#),
+                "tail" => Node::Builtin(r#"
+                    forall [t0] (
+                        <-tables: [t0],
+                        n: int,
+                        ?offset: int
+                    ) -> [t0]
+                "#),
+                "time" => Node::Builtin("forall [t0] (v: t0) -> time"),
+                "timeShift" => Node::Builtin(r#"
+                    forall [t0] (
+                        <-tables: [t0],
+                        "duration": duration,
+                        ?columns: [string]
+                    ) -> [t0]
+                "#),
+                "tripleExponentialDerivative" => Node::Builtin(r#"
+                    forall [t0] where t0: Numeric, t1: Row (
+                        <-tables: [{_value: t0 | t1}],
+                        n: int
+                    ) -> [{_value: float | t1}]
+                "#),
+                "true" => Node::Builtin("forall [] bool"),
+                "uint" => Node::Builtin("forall [t0] (v: t0) -> uint"),
+                // This is suspicous: each input table stream is not restricted to the same schema.
+                // What's the right signature?
+                "union" => Node::Builtin(r#"
+                    forall [t0] where t0: Row, t1: Row (
+                        tables: [[t0]]
+                    ) -> [t1]
+                "#),
+                "unique" => Node::Builtin(r#"
+                    forall [t0] where t0: Row (
+                        <-tables: [t0],
+                        ?column: string
+                    ) -> [t0]
+                "#),
+                "window" => Node::Builtin(r#"
+                    forall [t0] where t0: Row, t1: Row (
+                        <-tables: [t0],
+                        ?every: duration,
+                        ?period: duration,
+                        ?offset: duration,
+                        ?timeColumn: string,
+                        ?startColumn: string,
+                        ?stopColumn: string,
+                        ?createEmpty: bool
+                    ) -> [t1]
```

Comment:
Why does window produce different output rows?

Reply:
In this case it's only because we add the `startColumn` and the `stopColumn` to the output if they don't already exist.

So do you think we should be removing most of the `column` arguments from our functions and just expect the defaults?  Stuff like this really hurts type inference.

Reply:
Ah that's right. In that case we should either define window to expect a `_start` and `_stop` column, or just add them as `range` does. And yes, also like range, we should remove the `column` arguments and specify the expected labels via the type signature. We should probably create an umbrella issue for all such identifiers.

Reply:
I created this issue: https://github.com/influxdata/flux/issues/2255

There are so many functions like this, I'm not goint to link to it for every example of this.

Reply:
That works for me. As long as we have an issue for it. Thank you.